docker/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.11.3-alpine
```

Comment:
Could we switch to the chainguard python images here? https://github.com/chainguard-images/images/tree/main/images/python#usage

Reply:
I can't see why a dockerized CLI application needs chainguard images, but I can try to switch. I need more time to experiment.

Reply:
@samj1912 there is no `git` inside `cgr.dev/chainguard/python` image, but it's required. So what about using `cgr.dev/chainguard/python-dev` version (since `cruft` it is a dev tool)

Reply:
also need to mention: `linux/arm/v6`, `linux/arm/v7` and `linux/386` are not supported by `cgr.dev/chainguard/python` image